Formula/llvm@7.rb

```diff
@@ -20,19 +20,21 @@ class LlvmAT7 < Formula
 
   # https://llvm.org/docs/GettingStarted.html#requirement
   depends_on "cmake" => :build
+  depends_on maximum_macos: [:catalina, :build]
   depends_on xcode: :build
   depends_on arch: :x86_64
-  depends_on "libffi"
+
+  uses_from_macos "libedit"
+  uses_from_macos "libffi", since: :catalina
+  uses_from_macos "libxml2"
+  uses_from_macos "ncurses"
+  uses_from_macos "zlib"
```

Comment:
This should be split into a separate pull request. If there's issues building, ping me and I will look into it.